Makefile

```diff
@@ -91,12 +92,21 @@ $(BUILD_DIR)/$(CONTAINERD_BIN): $(SOURCES) $(PLUGIN_SOURCES)
 		-gcflags '$(GO_GCFLAGS)' \
 		$(PROJECT)/cmd/containerd
 
+ifeq ($(ARCH),s390x)
```

Comment:
probably don't need this block other than tabs don't see a difference

Reply:
@mikebrow This block was added to execute unit tests without `-race` option on s390x architecture, because for unit test execution `-race` is not supported on linux/s390x.

Reply:
ok thx eyes skimmed right over that...

Should probably have an ENV for RACE_TEST_OPTION supporting race for linux/amd64, linux/ppc64le, linux/arm64, freebsd/amd64, netbsd/amd64, darwin/amd64, and windows/amd64 or not otherwise.

Reply:
You could copy what we have in containerd core `Makefile` and `Makefile."OS"` include; allows certain arches to turn off `-race` without any changes in the `Makefile` directly.  For *GOOS == Linux* we only turn on `-race` for `amd64`: https://github.com/containerd/containerd/blob/master/Makefile.linux#L28-L31

Reply:
sounds good Phil...